e2e/script.js

```diff
@@ -1,5 +1,27 @@
+/*
+Forest Explorer E2E Test Script
+==============================
+
+This script uses k6 browser to automate e2e tests for the Forest Explorer faucet.
+
+Test Flow:
+- For each page defined in config.js:
+  - Navigate to the page and check for a 200 response
+  - For each button: check existence, visibility, enabled state, and perform the expected action
+  - For each link: check existence, visibility, and validity
+  - Check that footer links are present and valid
+- For each claim scenario in config.js:
+  - Enter each address in the input field and click the claim button
+  - Check for success (transaction container appears) or error ("Invalid address" message)
+
+Notes:
+- All checks use k6's check() function for assertions
+- The script is designed to fail fast if any check does not pass
+- Update config.js to add or modify test scenarios as the UI evolves
```

Comment:
This can get outdated pretty quickly. I'd rather we have inline comments rather than a lengthy introduction.